packages/twenty-front/src/modules/ui/layout/tab/components/Tab.tsx

```diff
@@ -92,8 +96,24 @@ export const Tab = ({
       to={to}
     >
       <StyledHover>
-        {logo && <StyledLogo src={logo} alt={`${title} logo`} />}
-        {Icon && <Icon size={theme.icon.size.md} />}
+        <StyledIconContainer>
+          {logo && (
+            <Avatar
+              avatarUrl={logo}
+              size="md"
+              placeholder=""
+              iconColor={iconColor}
+            />
+          )}
+          {Icon && (
+            <Avatar
+              Icon={Icon}
+              size="md"
+              placeholder=""
```

Comment:
from Avatar definition

```
const placeholderChar = placeholder?.[0]?.toLocaleUpperCase();
  const showPlaceholder =
    noAvatarUrl || invalidAvatarUrls.includes(avatarImageURI);
(...)
{Icon ? (
        <Icon
          color={iconColor ? iconColor : 'currentColor'}
          size={theme.icon.size.xl}
        />
      ) : showPlaceholder ? (
        placeholderChar
      ) : (
        <StyledImage src={avatarImageURI} onError={handleImageError} alt="" />
      )}
```

I fear we end up dispaying 'undefined' ?

Reply:
fixed! realized it after grep's review :)